Sources/TSCTestSupport/SwiftTestingHelpers.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Testing
+
+import TSCBasic
+
+public func assertFileExists(_ path: AbsolutePath) {
```

Comment:
I would recommend plumbing down a `sourceLocation` argument here so that the diagnostics are attributed to the test, not the helper function.

Reply:
This is completed.